.github/workflows/tact.yml

```diff
@@ -22,6 +22,27 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
+      - name: Setup Node.js 18 for backwards compat tests
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          # without caching
+
+      - name: Backwards compat tests
+        run: |
+          # Temporarily ignore engines
+          yarn config set ignore-engines true
+          # Install dependencies, gen and build the compiler
+          yarn install
+          yarn clean
+          yarn gen
+          yarn build
+          # Test some specific things for backwards compatibility
+          yarn jest -t 'isSubsetOf'
```

Comment:
nit: I'd rather create a "backCompat" folder or something. It's odd to have such a specific thing in CI code.

Reply:
Hmm, good point, although my idea is to keep such list of things small up until Node.js 22 [goes into maintenance LTS state](https://nodejs.org/en/about/previous-releases) by the end of October 2025. This semi-arbitrary requirement is just for users to catch up on Node.js 22+ and not stumble on `.isSubsetOf is not a function` errors. Therefore, a whole new folder seems like fostering usage of features from v22+, which is opposite to the goal.

Reply:
Now that you mention it, I realized we have `"lib": ["ESNext"]` in `tsconfig.json`, and probably we could prevent any further issues by downgrading it to the lowest possible version of standard library. Can't use a method from too new of a library if it's not there, can we? :)

I tried it before, and unfortunately the code has quite a few places where not-yet-standard (ESNext-only) features are used, so it might take a bit more effort.